core/impl/fail_fast_retry_strategy.cxx

```diff
@@ -15,9 +15,15 @@
  *   limitations under the License.
  */
 
+#include "couchbase/retry_action.hxx"
+#include "couchbase/retry_reason.hxx"
+#include "couchbase/retry_request.hxx"
```

Comment:
Should `<>` be used for these includes?

Reply:
yes, in general I try to follow it as a style rule with <> for our "public API" and double quotes for the other parts, thanks for spotting it.